Composer/packages/extensions/obiformeditor/src/Form/ObjectFieldTemplate/styles.css

```diff
@@ -9,7 +9,6 @@
 
 .ObjectItem .ObjectItemField {
   flex: 1;
-  overflow: hidden;
```

Comment:
This seems a little risky to me. Does this cause any other issues in the form? cc @tdurnford

Reply:
@a-b-r-o-w-n  I have checked some cases, it does not cause any issue. I just don't remember why to add this property= =!